### PR TITLE
Stubbing out the CRUD functionality for apps and pods

### PR DESCRIPTION
**Pull request by jlaswell**

### CODE REVIEW

File: resources/pod.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package resources
+
+// Pod is used to relate to Kubernetes pods
+type Pod struct {
+}
```
Comment (kyleferguson) [inline resources/pod.go:6]: You can add `Name` to the pod, as that's the only data we'll be tracking initially

**Review by kyleferguson — DISMISSED** [review]
Pod struct change and 👍 from me

File: resources/pod.go (hunk 1 above)
Comment (jlaswell) [reply resources/pod.go:6]: Done

**Comment by jlaswell** [comment]
@kyleferguson updated

**Comment by kyleferguson** [comment]
lgtm when CI passes

**Pull request merged**